build_tools/run_examples.sh

```diff
@@ -5,7 +5,7 @@ set -e -x
 set -o pipefail
 CMD="jupyter nbconvert --to notebook --inplace --execute --ExecutePreprocessor.timeout=600"
 
-for notebook in examples/*.ipynb; do
+for notebook in examples/*method.ipynb; do
```

Comment:
That change is probably not intended?
```suggestion
for notebook in examples/*.ipynb; do
```